### ISSUE TITLE
unrecognized option '--fastq'

### ISSUE THREAD

**Issue by aakashsur** [bcgsc/arcs#147]
When running - 
```
arcs-make arcs-long \
    draft=test_23 \
    reads=reads \
    t=40 \
    c=4 \
    l=4 \
    a=0.5 
```
I get the following error - 
```
sh -c 'gunzip -c reads.fa.gz | /mmfs1/sw/contrib/stf-src/arcs/1.2.2/Examples/../Examples/long-to-linked-pe -l250 --fastq | pigz -p40 > reads.cut250.fq.gz'
/mmfs1/sw/contrib/stf-src/arcs/1.2.2/Examples/../Examples/long-to-linked-pe: unrecognized option '--fastq'
```
I assume this happens because my long reads are in reads.fa.gz rather than a fastq file. Per the docs, this should be okay, and the repo seems to have a newer version of the arcs-make file which might fix this. I'm currently using the latest tarball release, i.e. 1.2.2.

When I tried to compile from the repo, I got the following make error - 
```
Uncompress.cpp: In function ‘bool uncompress_init()’:
Uncompress.cpp:214:9: error: ‘HAVE_LIBDL’ was not declared in this scope; did you mean ‘HAVE_STDLIB_H’?
  214 |  return HAVE_LIBDL;
      |         ^~~~~~~~~~
      |         HAVE_STDLIB_H
make[2]: *** [Makefile:548: libcommon_a-Uncompress.o] Error 1
make[2]: Leaving directory '/mmfs1/sw/contrib/stf-src/arcs/arcs/Common'
make[1]: *** [Makefile:422: all-recursive] Error 1
make[1]: Leaving directory '/mmfs1/sw/contrib/stf-src/arcs/arcs'
make: *** [Makefile:342: all] Error 2
```

**Comment by aakashsur** [comment]
To confirm, updating to the unreleased arcs-make file does appear to the fix the issue.

**Comment by lcoombe** [comment]
Hi @aakashsur,

Thanks for the report - yes look like this was the issue that was fixed in https://github.com/bcgsc/arcs/pull/117.  It was an issue with the ARCS-long target in the Makefile. I'm glad that you got it working using the latest master branch!

I'll work on releasing a new ARCS version that incorporates that fix.

Thanks!
Lauren

**Comment by lcoombe** [comment]
Just an update that the ARCS-long target fix was included in release v1.2.3
